proto/internal/temporal/server/api/persistence/v1/task_queues.proto

```diff
@@ -39,6 +39,10 @@ message BuildId {
     // (-- api-linter: core::0142::time-field-type=disabled
     //     aip.dev/not-precedent: Using HLC instead of wall clock. --)
     temporal.server.api.clock.v1.HybridLogicalClock state_update_timestamp = 3;
+    // HLC timestamp representing when this build id was last made default in its version set.
+    // (-- api-linter: core::0142::time-field-type=disabled
+    //     aip.dev/not-precedent: Using HLC instead of wall clock. --)
+    temporal.server.api.clock.v1.HybridLogicalClock set_default_update_timestamp = 4;
```

Comment:
I think `became_default_timestamp` might be an easier to understand name

Reply:
Fair